planner/core/testdata/integration_suite_out.json

```diff
@@ -3196,47 +3196,44 @@
       {
         "SQL": "explain format = 'brief' select count(*) from fact_t join d1_t on fact_t.d1_k > d1_t.d1_k",
         "Plan": [
-          "HashAgg 1.00 root  funcs:count(Column#12)->Column#11",
```

Comment:
Expected, model2 prefers to use StreamAgg instead of HashAgg if no much data to process.